nixos/modules/security/acme.nix

```diff
@@ -77,6 +77,7 @@ let
 
     unitConfig = {
       ConditionPathExists = "!/var/lib/acme/.minica/key.pem";
+      StartLimitIntervalSec = 0;
```

Comment:
Can we get c31b03a92021e1bf3786573f51dd90ea50bbd53a as a separate PR?

Reply:
For those wondering - Mic92 did this in #147834 :)